scripts/build.sh

```diff
@@ -14,7 +14,7 @@ fi
 
 count=0
 while [[ 3 -gt $count ]]; do
-    docker build . -f scripts/Dockerfile.build -t acceptance-tests-"${TRIM_JOB_NAME}""${BUILD_NUMBER}"
+    docker build . -f scripts/Dockerfile.jenkins -t acceptance-tests-"${TRIM_JOB_NAME}""${BUILD_NUMBER}"
```

Comment:
Is there any reason for changing the extension?

Reply:
actually i have added a new dockerfile , once its actually a different build environment , instead of playing with conditionals and env vars, i created a dockerfike per env, one for "local "builds and another one for jenkins